src/web/backend/nextcloud/models.py

```diff
@@ -0,0 +1,21 @@
+""" Models to work with ldap objects, operated by EDAP library """
+
+
+class User:
+    """ Edap user model """
+    def __init__(self, fqdn, uid, given_name, mail, surname, object_class=None):
+        self.fqdn = fqdn
+        self.uid = uid
+        self.given_name = given_name
+        self.mail = mail
+        self.object_class = object_class
```

Comment:
Ditto for the `object_class` - this is an implementation detail of the LDAP storage.